conf/test.config

```diff
@@ -23,6 +23,7 @@ params {
   input = "https://github.com/nf-core/test-datasets/raw/ampliseq/samplesheets/Samplesheet.tsv"
   metadata = "https://github.com/nf-core/test-datasets/raw/ampliseq/samplesheets/Metadata.tsv"
   dada_ref_taxonomy = "rdp=18"
+  dada_ref_species = "rdp=18"
```

Comment:
Since this is only a band-aid on a not required parameter I wait until you have removed it completely.

Reply:
Yes, working on it!